Source/CBLBlipReplicator.m

```diff
@@ -140,6 +139,12 @@ - (void) databaseClosing {
 @synthesize savingCheckpoint=_savingCheckpoint, active=_active;
 #endif
 
+- (CBLCookieStorage*) cookieStorage {
```

Comment:
I don't think it's worth caching our own reference — it's easy enough to call `_db.cookieStorage` instead of `self.cookieStorage`.